src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx

```diff
@@ -1,33 +1,132 @@
+        <div>
+          {member.firstName} {member.lastName}
+        </div>
+      </Link>
+      <div className="projects__active--input">
+        {member.active ? (
+          <tasks className="isActive">
+            <i className="fa fa-circle" aria-hidden="true"></i>
+          </tasks>
+        ) : (
+          <div className="isNotActive">
+            <i className="fa fa-circle-o" aria-hidden="true"></i>
           </div>
-          <div>{member._id}</div>
+        )}
+      </div>
+      <div>{member._id}</div>
+    </div>
+  ));
+
+  allMemberTable = allMemberList.slice(skip, skip + take).map((member, index) => (
+    <div className="project-member-table-row" id={'tr_' + member._id} key={'al_' + member._id}>
+      <div>
+        <div>{skip + index + 1}</div>
+      </div>
+      <Link to={`/userprofile/${member._id}`} title="View Profile">
+        <div>
+          {member.firstName} {member.lastName}
         </div>
-      ));
-    }
-  }
+      </Link>
+      <div className="projects__active--input">
+        {member.active ? (
+          <tasks className="isActive">
+            <i className="fa fa-circle" aria-hidden="true"></i>
+          </tasks>
+        ) : (
+          <div className="isNotActive">
+            <i className="fa fa-circle-o" aria-hidden="true"></i>
+          </div>
+        )}
+      </div>
+
+      <div>{member._id}</div>
+    </div>
+  ));
+
   return (
     <div className="project-member-table">
       <h5 className="project-member-table-title">Members</h5>
+      <div className="project-member-count-head">
+        <div
+          onChange={e => {
+            setMemberFilter(e.target.value);
+          }}
+        >
+          <input type="radio" name="memberFilter" value="active" id="active" defaultChecked />
+          <label htmlFor="active" id="project-active-member-count" className="project-member-count">
+            ACTIVE: {foundUsers.length}
+          </label>
+          <input type="radio" name="memberFilter" value="all-time" id="all-time" />
+          <label htmlFor="all-time" id="project-all-member-count" className="project-member-count">
+            ALL-TIME: {members.length}
+          </label>
+          {memberFilter == 'all-time'
+            ? handleMemberCount(allMemberList.length)
+            : handleMemberCount(activeMemberList.length)}
+        </div>
+      </div>
       <div className="project-member-table-row reports-table-head">
         <div id="projects__order">#</div>
         <div>Name</div>
+        <div>Active</div>
         <div>ID</div>
       </div>
-      <div>{memberList.length > 0 ? memberList : <Stub />}</div>
+      <div>
+        {memberFilter == 'all-time' ? (
+          allMemberTable.length > 0 ? (
+            allMemberTable
+          ) : (
+            <Stub />
+          )
+        ) : activeMemberTable.length > 0 ? (
+          activeMemberTable
+        ) : (
+          <Stub />
+        )}
```

Comment:
I'm not familiar with what the `Stub` component does.

I was wondering since an empty array would render nothing could this just become `{memberFilter === 'all-time' ? allMemberTable : activeMemberTable}`